### ISSUE TITLE
[Yetiforce-mailing] workflow, send inovice pdf

### ISSUE THREAD

**Issue by mariuszkrzaczkowski** [YetiForceCompany/YetiForceCRM#81]
in yeti 1.1 there is an error with workflow send pdf.

When I create a workflow and select "task" for worfkflow: SEND PDF
There are 3 dropdown boxes:

1. Select PDF template (OK)
2. MAIL_TPL_LIST => Select value (empty list)
3. Select e-mail address (default value is (Opportunities) All Comments
if value is changed to "(Users) Primary email" or "(Contacts) Primary email" value is not stored correct. And refresh page then show: "(Opportunities) All Comments" as selected email.

In other words it is not possible to schedule any pdf emails.

Maybe this is related to the fact that it is not possible to select module "invoice" in the "Mail templates" module picklist?

**Comment by rskrzypczak** [comment]
This functionality works properly. It is required to fil in all fields for workflow settings.

**Comment by rskrzypczak** [comment]
This functionality works properly. It is required to fil in all fields
for workflow settings. via 27686b1 [1] 

Links:
------
[1]
https://github.com/YetiForceCompany/YetiForceCRM/commit/27686b1b4f86381175240f92d1cdeb77652b6de4
